### PR TITLE
QOL: Build If Not Exist

### PR DESCRIPTION
**Pull request by keithjstone**
For images which are not expected to change. Don't rebuild if already built.

### CODE REVIEW

File: op.yml [hunk 1]
```diff
@@ -81,10 +86,19 @@ run:
         /var/run/docker.sock: dockerSocket
       envVars:
         imageName: $(imageName)
+        build: $(build)
       cmd:
         - sh
         - -ce
         - |
+          if [ "$build" = false ]; then \
```
Comment (chrisdostert) [inline op.yml:94]: One thing you can run into is the image may exist but would change if you re-built (either files in your context have changed or base image you depend on etc...). Building every time may introduce some time to the build but does ensure your resulting image is up to date.

Comment (keithjstone) [reply op.yml:94]: Yeah - I tried to model this after the same flag in docker compose which has the same drawbacks you describe (although I think does a deeper job of looking for changes in the docker file from the last run). I could put that warning in the description of the flag to make that tradeoff clear to users?

Comment (chrisdostert) [reply op.yml:94]: Just my personal opinion, but the juice doesn't seem worth the squeeze in that this is so infrequently run, I'd rather just have assurance we know It's right. If you feel strongly though I'll approve it.

Comment (keithjstone) [reply op.yml:94]: I"m ending up solving this by just building the images outside of opctl so I'll close this one.

**Pull request closed without merging**